Cdn77CodingStandard/Sniffs/Classes/ClassStructureSniff.php

```diff
@@ -1,169 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Cdn77CodingStandard\Sniffs\Classes;
-
-use PHP_CodeSniffer\Files\File;
-use PHP_CodeSniffer\Sniffs\Sniff;
-use PHP_CodeSniffer\Util\Tokens;
-use SlevomatCodingStandard\Helpers\TokenHelper;
-use const T_CONST;
-use const T_FUNCTION;
-use const T_PRIVATE;
-use const T_PROTECTED;
-use const T_PUBLIC;
-use const T_STRING;
-use const T_USE;
-use const T_VAR;
-use const T_VARIABLE;
-use function sprintf;
-use function strtolower;
-
-/**
- * This sniff ensures that the class/interface/trait has consistent order of its members in this exact order:
- *  - use statements
- *  - constants
- *  - properties
- *  - constructor
- *  - destructor
- *  - methods
- *  - magic methods
- */
-class ClassStructureSniff implements Sniff
```

Comment:
I was using this sniff and I cannot find it in doctrine nor in slevomat coding style. So this has no alternative?

Reply:
It had some issues. No replacement now but I'm planning on including one soon.

Reply:
Implemented in https://github.com/cdn77/coding-standard/commit/33b7002d2a2cc45907aa7293816b49da982fdfa3